### PR TITLE
Remove PROTOTYPEs from highlighting

### PR DESCRIPTION
**Pull request by nik9000**
Relates to #17085

### CODE REVIEW

**Comment by nik9000** [comment]
Should be a simple one.

**Comment by nik9000** [comment]
I spoke too soon. Looks like this breaks TopHits....

**Comment by nik9000** [comment]
I believe I just fixed TopHits.

File: core/src/main/java/org/elasticsearch/search/aggregations/metrics/tophits/TopHitsAggregatorBuilder.java [hunk 1]
```diff
@@ -538,7 +538,7 @@ protected TopHitsAggregatorBuilder doReadFrom(String name, StreamInput in) throw
             factory.fieldNames = fieldNames;
         }
         factory.from = in.readVInt();
-        in.readOptionalWriteable(HighlightBuilder::new);
+        factory.highlightBuilder = in.readOptionalWriteable(HighlightBuilder::new);
```
Comment (javanna) [inline core/src/main/java/org/elasticsearch/search/aggregations/metrics/tophits/TopHitsAggregatorBuilder.java]: haha I remember making the same mistake at some point :)

**Comment by javanna** [comment]
LGTM

**Pull request merged**